ecllibrary/std/Uni.ecl

```diff
@@ -398,4 +398,17 @@ EXPORT unicode GetNthWord(unicode text, unsigned4 n, varstring localename = '')
 EXPORT ExcludeNthWord(unicode text, unsigned4 n, varstring localename = '') :=
     lib_unicodelib.UnicodeLib.UnicodeLocaleExcludeNthWord(text, n, localename);
 
+/**
+ * Returns the source string with the all characters that match characters in the search string replaced
+ * with the character at the corresponding position in the replacement string.
```

Comment:
The code doesn't try to recognize that. Instead, the search and replacement strings will get passed to the TRANSLATE function where it will first check to make sure that the two strings (now made into Unicode strings) equal each other in character length (character length as in number of code points, not units). I'm not sure how I would check to see if a lone surrogate would bypass this or cause problems.